pulp_docker/app/viewsets.py

```diff
@@ -148,42 +149,40 @@ def sync(self, request, pk):
         return OperationPostponedResponse(result, request)
 
 
-class DockerPublisherViewSet(PublisherViewSet):
+class DockerPublicationViewSet(NamedModelViewSet, mixins.CreateModelMixin):
     """
-    A ViewSet for DockerPublisher.
+    A ViewSet for Docker Publication.
     """
 
-    endpoint_name = 'docker'
-    queryset = models.DockerPublisher.objects.all()
-    serializer_class = serializers.DockerPublisherSerializer
+    endpoint_name = 'docker/publications'
```

Comment:
I think this endpoint should be publications/docker. This is closer to the truth, the object is created *is* a publication, and the returned value of a /v3/publication will be less surprising.

Its worth noting that all the master/detail endpoints are structured in this way, ie content/docker remotes/docker